### PR TITLE
Updating code owners for golden file changes

### PR DESCRIPTION
**Pull request by Piinks**
## Description

Adding myself to the code owners for golden file changes

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 #
 # These names are just suggestions. It is fine to have your changes
 # reviewed by someone else.
 #
 # Use git ls-files '<pattern>' without a / prefix to see the list of matching files.
 
-/CODEOWNERS @magder
-/packages/flutter_tools/templates/module/ios/ @magder
-/packages/flutter_tools/templates/**/Podfile* @magder
+/bin/internal/goldens.version @Piinks
```
Comment (jmagman) [inline CODEOWNERS]: Do you want to include /dev/bots/download_goldctl.sh, /packages/flutter_goldens, etc?  Or will this goldens.version always be bumped with those changes?

Comment (Piinks) [reply CODEOWNERS]: For now, it'll always bump. :) 
I would like to keep a close eye on it while changing over to the new system that won't use this anymore.

**Review by jmagman — APPROVED** [review]
LGTM!
Thanks for fixing up my handle and let's ever discuss it again...

File: CODEOWNERS [hunk 2]
```diff
@@ -1,11 +1,12 @@
-# Below is a list of Flutter team members who are suggested reviewers
-# for contributions to this repository.
+# Below is a list of Flutter team members' Github handles who are
```
Comment (jmagman) [inline CODEOWNERS]: Oh also can you change to "GitHub" casing for consistency?

**Pull request merged**